modules/utxo-bin/package.json

```diff
@@ -29,7 +29,7 @@
     "@bitgo/blockapis": "^1.10.6",
     "@bitgo/statics": "^50.8.0",
     "@bitgo/utxo-lib": "^11.0.1",
-    "@bitgo/unspents": "^0.47.13",
+    "@bitgo/unspents": "^0.47.14",
```

Comment:
If you are updating the dependency, please add `package-lock.json` file as well.

Reply:
As discussed with @venkateshv1266, the `package-lock.json` is not required as it is an [untracked file](https://github.com/BitGo/BitGoJS/blob/1fb05dfecd51bc2916fa2d13111afde81ed89935/.gitignore)